src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
-import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Prop = {
+  moviesList: Movie[];
+};
+export const MoviesList: React.FC<Prop> = ({ moviesList }) => (
+  <div className="movies">
+    {moviesList.map(movie => (
+      <MovieCard
+        key={movie.imdbId}
+        title={movie.title}
+        description={movie.description}
+        imgUrl={movie.imgUrl}
+        imdbUrl={movie.imdbUrl}
+      />
```

Comment:
не перечисляй каждое свойство, передай объект целеком
```suggestion
      <MovieCard
        key={movie.imdbId}
        movie={movie}
      />
```